functions/src/monitor/monitor.ts

```diff
@@ -41,8 +44,18 @@ export const getMonitorPings: any = async (projectUid: string, monitorUid: strin
 
   await FirebaseAdmin
     .firestore()
-    .collection(`projects/${projectUid}/pings`)
-    .add(pingResult);
+    .collection(`projects/${projectUid}/${monitorUid}`)
+    .doc(uid)
+    .set(pingResult);
 
   return pingResult;
 }
+
+export const deleteMonitorPings: any = async (projectUid: string, monitorUid: string) => {
+  const documents: DocumentReference[] = await FirebaseAdmin.firestore()
+    .collection(`projects/${projectUid}/${monitorUid}`)
+    .listDocuments()
+  documents.forEach((doc: any) => {
```

Comment:
Should the type here be a `PingModel`

Reply:
PingModel is not application for doc type here. Please provide your inputs on it.